src/test/java/racingcar/RandomNumberGeneratorTest.java

```diff
@@ -0,0 +1,18 @@
+package racingcar;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import racingcar.domain.NumberGenerator;
+import racingcar.domain.RandomNumberGenerator;
+
+public class RandomNumberGeneratorTest {
+
+    @Test
+    void generate는_0_9사이의_랜덤숫자를_생성한다() {
+        NumberGenerator randomNumberGenerator = new RandomNumberGenerator();
+        for (int i = 0; i < 1000; i++) {
+            assertThat(randomNumberGenerator.generate()).isBetween(0, 9);
+        }
+    }
```

Comment:
Junit5의  @RepeatedTest 애너테이션을 사용하여 테스트를 반복 실행할 수도 있습니다 !

```suggestion
    @RepeatedTest(1000) // 1000번 반복
    void generate는_0_9사이의_랜덤숫자를_생성한다() {
        NumberGenerator randomNumberGenerator = new RandomNumberGenerator();
        assertThat(randomNumberGenerator.generate()).isBetween(0, 9);
    }
```

Reply:
오호 재밌는 어노테이션이군요! 이상하고 신비한 테스트 나라네요🥳🥳